app/controllers/tags_controller.rb

```diff
@@ -0,0 +1,32 @@
+class TagsController < ApplicationController
+  def index
+    @tags = Tag.all
+  end
+
+  def show
+    @tag = Tag.find(params[:id])
+    @questions = @tag.questions
+  end
+
+  def find
+    query = params[:q]
+    @tag_results = Tag.where("name like '%{query}'").map do |tag|
+      { name: tag.name }
+    end
+
+    content_type :json
+    @tag_results.to_json
```

Comment:
look at the docs for `.to_json` it takes arguments that all you to specify which attributes you want to have included in the json.